### PR TITLE
add: multi-platform console backed logger

### PR DESCRIPTION
**Pull request by hearnadam**
- move SL4J logger to JVM specific file

This should be an improved experience as users can explicitly rely on console logger if they so choose.

### CODE REVIEW

File: kyo-core/shared/src/main/scala/kyo/logs.scala [hunk 1]
```diff
@@ -58,53 +55,57 @@ object Logs:
...
 
     object Unsafe:
-        def apply(logger: org.slf4j.Logger): Unsafe = new Unsafe:
-            inline def traceEnabled: Boolean = logger.isTraceEnabled
-            inline def debugEnabled: Boolean = logger.isDebugEnabled
-            inline def infoEnabled: Boolean  = logger.isInfoEnabled
-            inline def warnEnabled: Boolean  = logger.isWarnEnabled
-            inline def errorEnabled: Boolean = logger.isErrorEnabled
+        class ConsoleLogger(name: String) extends Logs.Unsafe:
+            inline def traceEnabled: Boolean = true
```
Comment (hearnadam) [inline kyo-core/shared/src/main/scala/kyo/logs.scala:59]: I figured if this is a class, users can extend and override these values.

File: kyo-core/shared/src/main/scala/kyo/logs.scala [hunk 2]
```diff
@@ -58,53 +55,57 @@ object Logs:
...
+            inline def infoEnabled: Boolean = true
+
+            inline def warnEnabled: Boolean = true
+
+            inline def errorEnabled: Boolean = true
 
             inline def trace(msg: => String)(
                 using file: internal.Position
-            ): Unit = if traceEnabled then logger.trace(s"[$file] $msg")
+            ): Unit = if traceEnabled then println(s"TRACE $name -- [$file] $msg")
```
Comment (hearnadam) [inline kyo-core/shared/src/main/scala/kyo/logs.scala:71]: I tried to follow the SLF4J format I saw in the tests (without the cost of clock or thread id)

`21:02:54.003 [scala-execution-context-global-30] WARN kyo.logs -- [logsTest.scala:35] warn`

**Comment by fwbrasil** [comment]
Nice cleanup! I haven't been very disciplined creating the JS stubs :)

File: kyo-core/shared/src/main/scala/kyo/logs.scala (hunk 1 above)
Comment (fwbrasil) [reply kyo-core/shared/src/main/scala/kyo/logs.scala:59]: Nice!!

**Pull request merged**